From a8a6c65facfed6baffa922fcb7b87b310f5e2cba Mon Sep 17 00:00:00 2001
From: Balamurugan Ramar <quic_bramar@quicinc.com>
Date: Wed, 2 Apr 2025 17:40:41 +0530
Subject: [PATCH 08/25] makefile: Add wireless driver for kernel build

This change enables the inclusion of wireless drivers
specified by CPTCFG_WLAN in the kernel build process

Signed-off-by: Balamurugan Ramar <quic_bramar@quicinc.com>
Signed-off-by: Kaviarasu Sanjeevi <quic_kaviaras@quicinc.com>
---
 Makefile.kernel | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.kernel b/Makefile.kernel
index ec0a886..5da6e30 100644
--- a/Makefile.kernel
+++ b/Makefile.kernel
@@ -41,7 +41,7 @@ obj-$(CPTCFG_MAC80211) += net/mac80211/
 #obj-$(CPTCFG_QRTR) += net/qrtr/
 #obj-$(CPTCFG_QCOM_QMI_HELPERS) += drivers/soc/qcom/
 #obj-$(CPTCFG_MHI_BUS) += drivers/bus/mhi/
-#obj-$(CPTCFG_WLAN) += drivers/net/wireless/
+obj-$(CPTCFG_WLAN) += drivers/net/wireless/
 #obj-$(CPTCFG_USB_NET_RNDIS_WLAN) += drivers/net/usb/
 #
 #obj-$(CPTCFG_USB_WDM) += drivers/usb/class/
-- 
